youtube_dl/extractor/indavideo.py

```diff
@@ -9,8 +11,14 @@
 )
 
 
-class IndavideoEmbedIE(InfoExtractor):
```

Comment:
As said **don't touch** extractor's name. We don't need unnecessary download archive breakages.